doc/api/domain.markdown

```diff
@@ -308,56 +269,56 @@ than crashing the program.
 
 * {Array}
 
-An array of timers and event emitters that have been explicitly added
-to the domain.
+Returns an array of timers and `EventEmitter` objects that have been explicitly
+bound to the domain.
 
 ### domain.add(emitter)
 
 * `emitter` {EventEmitter | Timer} emitter or timer to be added to the domain
 
-Explicitly adds an emitter to the domain.  If any event handlers called by
-the emitter throw an error, or if the emitter emits an `'error'` event, it
-will be routed to the domain's `'error'` event, just like with implicit
-binding.
+Explicitly binds an `EventEmitter` or timer to the domain.  If any event
+handlers called by the emitter throw an error, or if the `EventEmitter` emits
+an `'error'` event, the error will be routed to the `Domain` objects own
+`'error'` event.
```

Comment:
I would suggest "will be routed to the `Domain`'s own `'error'` event handler.".